src/init.cpp

```diff
@@ -1545,6 +1545,17 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     }
 #endif // ENABLE_MINING
 
+    // Start the thread that notifies listeners of transactions that have been
+    // recently added to the mempool, or have been added to or removed from the
```

Comment:
I don't understand how transactions can be removed from the chain, unless there's been a fork.